htop.1.in

```diff
@@ -656,10 +656,12 @@ Memory sizes in
 .B htop
 are displayed in a human-readable form.
 Sizes are printed in powers of 1024. (e.g., 1023M = 1072693248 Bytes)
+If no suffix is shown the units are implicitly KiB.
```

Comment:
```suggestion
If no suffix is shown the units are implicitly K as in KiB (Kibibyte, 1 KiB = 1024 Bytes).
```

Reply:
@fasterit "bytes", when spelled in full, should be all lowercase, not title case.
Same goes for "kibibytes".